sentry-core/src/main/java/io/sentry/core/SentryEnvelopeItem.java

```diff
@@ -5,7 +5,7 @@
   // TODO: Can we have a slice or a reader here instead?
   private final byte[] data;
 
-  public SentryEnvelopeItem(SentryEnvelopeItemHeader header, byte[] data) {
+  SentryEnvelopeItem(SentryEnvelopeItemHeader header, byte[] data) {
```

Comment:
Doesn't this means this type can't be constructed outside of this package?
That means this isnt' just a refactor but it changes the API

Reply:
@bruno-garcia I only see it being used by `EnvelopeReader`, why would we expose it?

Reply:
Should we make the whole type internal?
Seems like we can get around with the whole Envelope thing internal for now.

Only once we have a JVM based integration we'd need to open this up.

Reply:
yep, right now as we have different packages over the modules, it might not be fully possible, maybe for Envelopes, but not for all the cases, we should work on that before the first release.